CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+4.63.0
+==========
+### New Features
+- Implemented Claims API to Bypass Cache When Claims are Present in MSAL with Managed Identity. This is still an experimental API and may change in the future. See [Issue #4845](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4845)
+
+### Bug Fixes
+- Fixed an issue where SHR POP tokens were broken due to an invalid algorithm. The algorithm has been corrected to PS256 from RS256. See [Issue #4839](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4839)
+- Addressed a problem with incorrectly encoded JWK format in RSA public key export. See [Issue #4881](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4881)
```

Comment:
Not sure we need to mention this one. This bug technically never was released right?